Error in Deployment Workflow

`CARGO_REGISTRY_TOKEN` is defined in wrong section.


```
Run cargo publish -p torrust-tracker-contrib-bencode
  cargo publish -p torrust-tracker-contrib-bencode
  cargo publish -p torrust-tracker-located-error
  cargo publish -p torrust-tracker-primitives
  cargo publish -p torrust-tracker-configuration
  cargo publish -p torrust-tracker-test-helpers
  cargo publish -p torrust-tracker
  shell: /usr/bin/bash -e {0}
  env:
    CARGO_INCREMENTAL: 0
    CARGO_TERM_COLOR: always
    Updating crates.io index
error: no token found, please run `cargo login`
or use environment variable CARGO_REGISTRY_TOKEN
Error: Process completed with exit code 101.
```

--- a/.github/workflows/deployment.yaml
+++ b/.github/workflows/deployment.yaml
@@ -64,11 +64,10 @@ jobs:
         with:
           toolchain: stable
 
-        env:
-          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
-
       - id: publish
         name: Publish Crates
+        env:
+          CARGO_REGISTRY_TOKEN: "${{ secrets.CARGO_REGISTRY_TOKEN }}"
         run: |
           cargo publish -p torrust-tracker-contrib-bencode
           cargo publish -p torrust-tracker-located-error

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -26,7 +26,7 @@ license-file = "COPYRIGHT"
 publish = true
 repository = "https://github.com/torrust/torrust-tracker"
 rust-version = "1.72"
-version = "3.0.0-alpha.7-develop"
+version = "3.0.0-alpha.8-develop"
 
 [dependencies]
 aquatic_udp_protocol = "0.8"
@@ -56,10 +56,10 @@ serde_json = "1.0"
 serde_with = "3.2"
 thiserror = "1.0"
 tokio = { version = "1.29", features = ["macros", "net", "rt-multi-thread", "signal", "sync"] }
-torrust-tracker-configuration = { version = "3.0.0-alpha.7-develop", path = "packages/configuration" }
-torrust-tracker-contrib-bencode = { version = "3.0.0-alpha.7-develop", path = "contrib/bencode" }
-torrust-tracker-located-error = { version = "3.0.0-alpha.7-develop", path = "packages/located-error" }
-torrust-tracker-primitives = { version = "3.0.0-alpha.7-develop", path = "packages/primitives" }
+torrust-tracker-configuration = { version = "3.0.0-alpha.8-develop", path = "packages/configuration" }
+torrust-tracker-contrib-bencode = { version = "3.0.0-alpha.8-develop", path = "contrib/bencode" }
+torrust-tracker-located-error = { version = "3.0.0-alpha.8-develop", path = "packages/located-error" }
+torrust-tracker-primitives = { version = "3.0.0-alpha.8-develop", path = "packages/primitives" }
 tower-http = { version = "0.4", features = ["compression-full"] }
 uuid = { version = "1", features = ["v4"] }
 
@@ -70,7 +70,7 @@ reqwest = { version = "0.11.18", features = ["json"] }
 serde_bytes = "0.11"
 serde_repr = "0.1"
 serde_urlencoded = "0.7"
-torrust-tracker-test-helpers = { version = "3.0.0-alpha.7-develop", path = "packages/test-helpers" }
+torrust-tracker-test-helpers = { version = "3.0.0-alpha.8-develop", path = "packages/test-helpers" }
 
 [workspace]
 members = ["contrib/bencode", "packages/configuration", "packages/located-error", "packages/primitives", "packages/test-helpers"]

--- a/README.md
+++ b/README.md
@@ -179,13 +179,13 @@ This project was a joint effort by [Nautilus Cyberneering GmbH][nautilus] and [D
 
 [containers.md]: ./docs/containers.md
 
-[api]: https://docs.rs/torrust-tracker/3.0.0-alpha.7-develop/torrust_tracker/servers/apis/v1
-[http]: https://docs.rs/torrust-tracker/3.0.0-alpha.7-develop/torrust_tracker/servers/http
-[udp]: https://docs.rs/torrust-tracker/3.0.0-alpha.7-develop/torrust_tracker/servers/udp
+[api]: https://docs.rs/torrust-tracker/3.0.0-alpha.8-develop/torrust_tracker/servers/apis/v1
+[http]: https://docs.rs/torrust-tracker/3.0.0-alpha.8-develop/torrust_tracker/servers/http
+[udp]: https://docs.rs/torrust-tracker/3.0.0-alpha.8-develop/torrust_tracker/servers/udp
 
 [good first issues]: https://github.com/torrust/torrust-tracker/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
 [documentation]: https://docs.rs/torrust-tracker/
-[API documentation]: https://docs.rs/torrust-tracker/3.0.0-alpha.7-develop/torrust_tracker/servers/apis/v1
+[API documentation]: https://docs.rs/torrust-tracker/3.0.0-alpha.8-develop/torrust_tracker/servers/apis/v1
 [discussions]: https://github.com/torrust/torrust-tracker/discussions
 
 [COPYRIGHT]: ./COPYRIGHT

--- a/docs/release_process.md
+++ b/docs/release_process.md
@@ -13,12 +13,12 @@ Torrust Tracker is published according to this protocol:
 
 - [ ] `develop` is ready for branching for a release.
 - [ ] force-push develop to `staging` branch.
-- [ ] commit `release: version (semantic version)`, removing the `-develop` suffix.
+- [ ] commit `release: version (semantic version)`, that removes the `-develop` suffix.
 - [ ] create pull request to merge `staging` into `main` branch.
 - [ ] check all status checks succeed for `main` branch.
-- [ ] push `main` branch to `releases\v(semantic version)` branch.
-- [ ] check all status checks success for `releases\v(semantic version)` branch.
-- [ ] create signed `v(semantic version)` tag from `releases\v(semantic version) HEAD`.
+- [ ] push `main` branch to `releases/(semantic version)` branch.
+- [ ] check all status checks success for `releases/(semantic version)` branch.
+- [ ] create signed `v(semantic version)` tag from `releases/(semantic version) HEAD`.
 - [ ] create github release from `v(semantic version)` tag.
 - [ ] merge the `main` branch back into `develop` branch, assuring that the (semantic version) has the suffix `-develop`.
 

--- a/packages/configuration/Cargo.toml
+++ b/packages/configuration/Cargo.toml
@@ -21,8 +21,8 @@ serde = { version = "1.0", features = ["derive"] }
 serde_with = "3.2"
 thiserror = "1.0"
 toml = "0.8"
-torrust-tracker-located-error = { version = "3.0.0-alpha.7-develop", path = "../located-error" }
-torrust-tracker-primitives = { version = "3.0.0-alpha.7-develop", path = "../primitives" }
+torrust-tracker-located-error = { version = "3.0.0-alpha.8-develop", path = "../located-error" }
+torrust-tracker-primitives = { version = "3.0.0-alpha.8-develop", path = "../primitives" }
 
 [dev-dependencies]
 uuid = { version = "1", features = ["v4"] }

--- a/packages/test-helpers/Cargo.toml
+++ b/packages/test-helpers/Cargo.toml
@@ -17,5 +17,5 @@ version.workspace = true
 [dependencies]
 lazy_static = "1.4"
 rand = "0.8.5"
-torrust-tracker-configuration = { version = "3.0.0-alpha.7-develop", path = "../configuration" }
-torrust-tracker-primitives = { version = "3.0.0-alpha.7-develop", path = "../primitives" }
+torrust-tracker-configuration = { version = "3.0.0-alpha.8-develop", path = "../configuration" }
+torrust-tracker-primitives = { version = "3.0.0-alpha.8-develop", path = "../primitives" }

